www/src/spec.js

```diff
@@ -78,10 +78,10 @@ export function Requirements({ requirements, showSection }) {
       field: "section",
       headerName: "Section",
       valueGetter(params) {
-        return params.row.section.id;
+        return params.row;
       },
-      sortComparator(v1, v2, row1, row2) {
-        return row1.row.cmp(row2.row);
+      sortComparator(v1, v2) {
+        return v2.cmp(v1);
```

Comment:
Should these be reversed?
```suggestion
        return v1.cmp(v2);
```

Reply:
Yeah, I was thinking most people would want desc. ordering on the first click since the list was already in asc. order from the get-go. Just realized there's a better way to do it.